pkg/controllers/vdb/onlineupgrade_reconciler.go

```diff
@@ -329,15 +328,17 @@ func (r *OnlineUpgradeReconciler) postQueryOriginalConfigParamDisableNonReplicat
 
 // queryOriginalConfigParamDisableNonReplicatableQueries will run the reconciler to get value of the config parameter
 // DisableNonReplicatableQueries at database level within main cluster
-func (r *OnlineUpgradeReconciler) queryOriginalConfigParamDisableNonReplicatableQueries(ctx context.Context) (ctrl.Result, error) {
+func (r *OnlineUpgradeReconciler) queryOriginalConfigParamDisableNonReplicatableQueries(ctx context.Context) (res ctrl.Result, err error) {
 	pf := r.PFacts[vapi.MainCluster]
-	actor := MakeGetConfigurationParameterReconciler(r.VRec, r.Log, r.VDB, pf, r.Dispatcher, r.VRec.Client,
-		ConfigParamDisableNonReplicatableQueries,
-		ConfigParamDisableNonReplicatableQueriesLevel,
-		&r.originalConfigParamDisableNonReplicatableQueriesValue, /*out param*/
-	)
-	r.Manager.traceActorReconcile(actor)
-	return actor.Reconcile(ctx, &ctrl.Request{})
+	initiator, ok := pf.findFirstUpPod(false /*not allow read-only*/, "" /*arbitrary subcluster*/)
+	if !ok {
+		r.Log.Info("No Up nodes found. Requeue reconciliation.")
+		return ctrl.Result{Requeue: true}, nil
+	}
+	vc := catalog.MakeVCluster(r.VDB, pf.VerticaSUPassword, initiator.podIP, r.Log, r.VRec.Client, r.VRec.EVRec)
+	r.originalConfigParamDisableNonReplicatableQueriesValue, err = vc.GetConfigurationParameter(ConfigParamDisableNonReplicatableQueries,
```

Comment:
Does that mean that the min server version that supports online upgrade is 24.4?
@cchen-vertica what should we do? The next release is 24.3 and it does not support `non replicatable` as @fenic-fawkes added a change to the server recently.